extensions/@shopgate-theme-config/frontend/config-common.js

```diff
@@ -43,6 +43,59 @@ export default {
             name: 'ShopgateProductOrderQuantityHint',
             id: '@shopgate/engage/product/OrderQuantityHint',
           },
+          {
+            name: 'ShopgateProductSwatch',
+            id: '@shopgate/engage/product/Swatch',
+            settings: {
+              selectionStyles: {
+                selected: {
+                  borderColor: undefined,
+                },
+                unselected: {
+                  borderColor: undefined,
+                },
+              },
+              elementWidth: '1.0rem',
```

Comment:
Can go to swatch styles

Reply:
I don't agree, because this is not a simple style. This value is spread across multiple styles.

Reply:
I've renamed the settings though to ìtemWidth` and `itemHeight` because the term `Element` is not in use anymore and item being more appropriate now,because they also are actually list items.